evap/staff/tests.py

```diff
@@ -1081,28 +1085,96 @@ def load_test_data(self):
 
 class MergeUsersTest(TestCase):
 
-    def test_merge_users(self):
-        course1 = mommy.make(Course)
-        course2 = mommy.make(Course)
-
-        delegate1 = mommy.make(UserProfile, pk=1)
-        delegate2 = mommy.make(UserProfile, pk=2)
-
-        user1 = mommy.make(UserProfile, pk=3, username="user1", course_set=[course1], delegates=[delegate1])
-        user2 = mommy.make(UserProfile, username="user2", course_set=[course2], delegates=[delegate2])
-        course2.voters = [user2]
-
-        self.assertEqual(UserProfile.objects.count(), 4)
-        call_command("merge_users", str(user1.pk), str(user2.pk), stdout=StringIO())
-        self.assertEqual(UserProfile.objects.count(), 3)
-
-        self.assertEqual(user1.pk, 3)
-        self.assertEqual(user1.username, "user1")
-        self.assertEqual(set(user1.course_set.all()), set([course1, course2]))
+    @classmethod
+    def setUpTestData(cls):
+        cls.user1 = mommy.make(UserProfile, username="test1")
+        cls.user2 = mommy.make(UserProfile, username="test2")
+        cls.user3 = mommy.make(UserProfile, username="test3")
+        cls.group1 = mommy.make(Group, name="group1")
+        cls.group2 = mommy.make(Group, name="group2")
+        cls.mainuser = mommy.make(UserProfile,
+            username="main",
+            title="Dr.",
+            first_name="Main",
+            last_name="",
+            email="main@test.com",
+            groups=[cls.group1],
+            delegates=[cls.user1, cls.user2],
+            represented_users=[cls.user3],
+            cc_users=[cls.user1],
+            ccing_users=[]
+        )
+        cls.mergeuser = mommy.make(UserProfile,
```

Comment:
could you call these `main_user` and `other_user` to be in line with the implementation?